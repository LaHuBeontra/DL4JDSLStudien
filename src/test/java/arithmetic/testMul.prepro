function main() {
    import vec3 p1, vec3 p2;

    vec3 mul = p1 * p2;

    export mul;
}
