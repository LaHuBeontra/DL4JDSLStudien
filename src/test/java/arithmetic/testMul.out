PreProDataSet{
dataSet=[1,806.00,  0.00,  0.00,  44,662.00,  0.00,  0.00,  30.00,  0.00,  0.00,  42.00,  0.00,  0.00]
variableNames=[mul]
variableShapes=[4, 3]}